source/content/oss-support-levels.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Pantheon Open Source Software Support Levels
+description: Learn about Pantheon Support Levels for Open Source Sofware
+categories: [get-started, oss]
+tags: [collaborate]
+---
+
+At Pantheon we love Open Source, we have created lots of public repositories in our [Github account](https://github.com/orgs/pantheon-systems/repositories) and we have started tagging them with support level badges. This page describes each of those levels and what should you expect when using those repositories.
+
+## Pilot
+
+A project in Pilot support level is a new product or feature set that we are working on. The code is publicly available but if there are any accompaning platform components you will need an invite through the project pilot program. Even though we make our best effort, there may be bugs and we need your help to identify them so that we can fix them. The product or feature set may not be feature complete and development is still going on. Support in this stage is provided by the Engineering team, and is only available to members of the program.
+
+## Early Acess
+
+Early Access projects are similar to Pilot projects, but are a little more mature, and have been opened up for use by a larger number of users. An invitation is still necessary to join an Early Access program.
+
+## Limited Availability
+
+A project in this support level is a product or feature set that is complete and can be used in all environments including production. Clients can use the new functionality without requiring an invite but it is an opt-in process. You are highly encouraged to report any bug you may find so that we can make a better product for everyone. Support in this stage is available through regular Pantheon channels.
+
+## Actively Maintained
+
+A project in this support level has been publicly launched and has several eyes over it ensuring that everything works as expected. Anyone could use this project and create issues and Pull Requests in the repository as needed and we will respond to them in a timely manner. Support in this stage is provided by the Customer Success Engineering team.
```

Comment:
@greg-1-anderson I'm guessing the same update should be made here, right? "Support in this stage is available through regular Pantheon channels."